modules/persistent-storage-csi-tp-enable.adoc

```diff
@@ -75,6 +79,19 @@ azure-disk-csi-driver-node-mqdhh                        3/3     Running 0
 azure-disk-csi-driver-operator-7d966fc6c5-x74x5         1/1     Running 0           44m
 ----
 endif::azure[]
+ifdef::alicloud[]
+[source, terminal]
+----
+NAME                                                    READY   STATUS  RESTARTS    AGE
+vmware-alicloud-csi-driver-controller-5646dbbf54-cnsx7   9/9     Running 0          4h29m
+vmware-alicloud-csi-driver-node-ch22q                    3/3     Running 0          4h37m
+vmware-alicloud-csi-driver-node-gfjrb                    3/3     Running 0          4h37m
+vmware-alicloud-csi-driver-node-ktlmp                    3/3     Running 0          4h37m
+vmware-alicloud-csi-driver-node-lgksl                    3/3     Running 0          4h37m
+vmware-alicloud-csi-driver-node-vb4gv                    3/3     Running 0          4h37m
+vmware-alicloud-csi-driver-operator-7c7fc474c-p544t      1/1     Running 0          4h29m
+----
+endif::alicloud[]
```

Comment:
TechPreviewNoUpgrade is not used in Alibaba, do we still need this chunk? Or any change in modules/persistent-storage-csi-tp-enable.adoc in general?

Reply:
This module is not actually included in the topic. The changes here got "caught" in this PR by accident.